specification/compute/resource-manager/Microsoft.Compute/stable/2018-06-01/compute.json

```diff
@@ -6230,6 +6230,7 @@
           "type": "object",
           "additionalProperties": {
             "type": "object",
+            "x-ms-client-name": "userAssignedIdentitiesValue",
```

Comment:
this is on old version, has this API version been releaesd in SDK already? If yes, specifying new client name will cause SDK breaking change right?

Reply:
Hi @yungezz , yes, the API version has been released.
The effect for this setting:
1. Will cause NO breaking-change for Track1 SDK. (since this set the same client-name as what the modeler1 can guess)
2. Will cause breaking change for Track2 SDK. (for fix issue found in https://github.com/Azure/azure-sdk-for-python/issues/14150) 

The root cause I submit this PR is the modeler4 guess different name with modeler1, which will cause the above issue in Track2.
CC to @jsntcy

Reply:
thanks @changlong-liu . For Track1, `this set the same client-name as what the modeler1 can guess` this is true for other language SDKs also right, since all SDKs based on modeler1? For Track2, only Python SDK consuming `x-ms-client-name`?  If both yes, I'll approve and merge the PR.

Reply:
Hi @yungezz , yes, This will make no change in any language SDK in Track1

Reply:
@changlong-liu how about Track2?

Reply:
Hi @yungezz, the x-ms-client-name is used by modeler4, so all language Track2 SDK will respect this setting. 

I guess all released Track2 SDKs will have problem similar to Azure/azure-sdk-for-python#14150.  So this is a breaking-change/bug-fix for all released track2 SDK.